src/models/cost_functions/ImportExportCost.jl

```diff
@@ -0,0 +1,288 @@
+"""
+$(TYPEDEF)
+$(TYPEDFIELDS)
+
+    ImportExportCost(import_offer_curves, export_offer_curves, energy_import_weekly_limit, energy_export_weekly_limits, ancillary_service_offers)
+    ImportExportCost(; import_offer_curves, export_offer_curves, energy_export_weekly_limits, ancillary_service_offers)
+
+An operating cost for imports/exports and ancillary services from neighboring areas. The data model
+employs a CostCurve{PiecewiseIncrementalCurve} with an implied zero cost at zero power.
+"""
+mutable struct ImportExportCost <: OperationalCost
+    "Buy Price Curves data to import power, which can be a time series of [`PiecewiseStepData`](@extref) or a
+    [`CostCurve`](@ref) of [`PiecewiseIncrementalCurve`](@ref)"
+    import_offer_curves::Union{
+        Nothing,
+        TimeSeriesKey,  # piecewise step data
+        CostCurve{PiecewiseIncrementalCurve},
+    }
+    "Sell Price Curves data to export power, which can be a time series of `PiecewiseStepData` or a
+    [`CostCurve`](@ref) of [`PiecewiseIncrementalCurve`](@ref)"
+    export_offer_curves::Union{
+        Nothing,
+        TimeSeriesKey,
+        CostCurve{PiecewiseIncrementalCurve},
+    }
+    "Weekly limit on the amount of energy that can be imported, defined in system base p.u-hours."
+    energy_import_weekly_limit::Float64
```

Comment:
Are we comfortable baking "weekly" into the struct itself? Do any other PSY struct fields specify a time duration in this way? What if someone wants to use something other than a week?

Reply:
we probably need to leave this for now since we can't really make models with uneven resolutions.